### ISSUE TITLE
Do you release the  object detection code?

### ISSUE THREAD

**Issue by qifeng22** [Pointcept/Pointcept#262]
I can't find object detection code. @Gofinge

**Comment by Gofinge** [comment]
Yes, as I still can not allocate some time to the code cleanup work. Object detection is built upon OpenPCDet, and the config is almost aligned with FlatFormer. if you are in a hurry in need of the code, you can contact me by email; I can share the raw code.

**Comment by yangfy0608** [comment]
I need the object detection code too, may i email you to ask for the raw code @Gofinge

**Comment by Gofinge** [comment]
Not much time for cleaning the code. Email me, and I can send the raw code.

**Comment by maxiuw** [comment]
hi @Gofinge, I sent you an email as well, it would be great if you could share the code with me as well!

**Comment by maxiuw** [comment]
@yangfy0608  @qifeng22  could you maybe share it with me ? maciejw@kth.se

**Comment by Gofinge** [comment]
> @yangfy0608 @qifeng22 could you maybe share it with me ? [maciejw@kth.se](mailto:maciejw@kth.se)

Sorry for missing the previous email. I just sent it. To busy for the CVPR deadline.

**Comment by yxchng** [comment]
@Gofinge Hi, I have also sent you an email as well. Can you kindly share the code? Thanks a lot.

**Comment by weilli** [comment]
@Gofinge Hi，I also sent you an email to apply for the code，thank you.

**Comment by Gofinge** [comment]
Hi all, sorry for the late response, I just recovered from the busy schedule for the CVPR project. I will gradually reply to all emails.

**Comment by BartvanMarrewijk** [comment]
Interested in that code as well. Even if the code is not fully cleaned, maybe nice to just put it under a detection branch?

**Comment by Gofinge** [comment]
> Interested in that code as well. Even if the code is not fully cleaned, maybe nice to just put it under a detection branch?

Absolutely

**Comment by Gofinge** [comment]
> Hi, I also sent you a request email for the object detection code. Thanks in advance.

Thanks for interesting about our research, I will check my mail box later today.

**Comment by david2611** [comment]
Hi, I have also sent a request for object detection code. Hope we aren't all bugging you too much :)
